src/main/java/services/gorest/actions/user/CreateUser.java

```diff
@@ -0,0 +1,34 @@
+package services.gorest.actions.user;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+import services.gorest.actions.GoRestActions;
+import services.gorest.models.User;
+import utils.reusable.specifications.ReusableSpecifications;
+
+public class CreateUser extends GoRestActions {
+
+    private String POST_USER_URL = getBaseUri() + USERS_ENDPOINT;
+
+    @Step("I create a new user")
+    public Response createNewUser(User user) {
+        Response response = SerenityRest.rest().given().log().all()
+                .spec(ReusableSpecifications.authorizedRequestSpec())
+                .baseUri(POST_USER_URL)
+                .contentType(ContentType.JSON)
+                .when()
+                .body(user)
+                .post();
+        response.then().log().all().spec(ReusableSpecifications.responseSpec());
```

Comment:
see the pic i attached. Sometime the call can hang and take more than 5 seconds. Is not fine to have this check on each call. You can do this if you specifically test the response time.